client/components/text-area/index.js

```diff
@@ -0,0 +1,59 @@
+import React, { PropTypes } from 'react';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormTextarea from 'components/forms/form-textarea';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormInputValidation from 'components/forms/form-input-validation';
+import { sanitize } from 'dompurify';
+
+const renderFieldDescription = ( description ) => {
```

Comment:
this is repeated across several components now, so prob an opportunity to DRY (can be a in a separate PR)